inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -69,8 +73,7 @@ def __init__(self,
             self.domain = domain
 
     def simulate(self,
-                 simulator: Simulator = OpenFOAM(
-                     "windtunnel.openfoam.run_simulation"),
+                 simulator: Simulator = OpenFOAM(),
```

Comment:
Was this change addressed in a backend, will this work?

Reply:
This works, on the backend just needs to be removed to reduce entropy.